src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -215,6 +215,12 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		navigationController?.topViewController?.view.endEditing(true)
 		navigationController?.pushViewController(vc, animated: true)
 	}
+	
+	private func present(_ vc: UIViewController, withNavigation: Bool = true) {
```

Comment:
```suggestion
	private func present(_ viewController: UIViewController, withNavigation: Bool = true) {
```

Reply:
Please be sure, that I made a fault before comment.
The shortcut is equal to previous implementation to be consistent. 
But I agree with you, I don't like that shortcuts as well. :)

```swift
// Current
private func push(_ vc: UIViewController) {
	navigationController?.topViewController?.view.endEditing(true)
	navigationController?.pushViewController(vc, animated: true)
}

private func present(_ vc: UIViewController, withNavigation: Bool = true) {
	let navVC = NavigationControllerWithLargeTitle(rootViewController: vc)
	navigationController?.topViewController?.view.endEditing(true)
	navigationController?.present(navVC, animated: true)
}
``` 

```swift
// Now
private func push(_ viewController: UIViewController) {
	navigationController?.topViewController?.view.endEditing(true)
	navigationController?.pushViewController(viewController, animated: true)
}

private func present(_ viewController: UIViewController, withNavigation: Bool = true) {
	let navVC = NavigationControllerWithLargeTitle(rootViewController: viewController)
	navigationController?.topViewController?.view.endEditing(true)
	navigationController?.present(navVC, animated: true)
}
```